deploy/docker-compose/README

```diff
@@ -0,0 +1,153 @@
+# MONAI Deploy Lite
+
+MONAI Deploy Lite provides a simple & lightweight solution for running MONAI Applications (MAPs) on a minimal MONAI Deploy platform.
+
+It is meant to bridge the big gap between local development of MAPs in a laptop using the MONAI Deploy App SDK, and a production-ready deployment. 
+
+Users of MONAI Deploy Lite will be able to run their MAPs, connected to a test PACS, or their own test / research PACS, for further validation, confidently taking steps towards production.   
+
+By reusing the same key core services for DICOM I/O and AI workflow orchestration, it provides the same functionality and consistent experience independently on where and how the applications are run, with minimal changes for the end user.
+
+
+## Prerequisites
+
+- [Docker Engine](https://docs.docker.com/engine/install/) v20.10.18+
+- [Docker Compose Plug-in](https://docs.docker.com/compose/install/) v2.10.2+
+- MONAI Deploy MAPs built using the [MONAI Deploy App SDK](https://github.com/Project-MONAI/monai-deploy-app-sdk)
+
+## Start the Services
+
+The docker [compose file](docker-compose.yml) spins up the following services. Each service may be accessed at the IP and port described below.
+
+Execute the following command to bring up all services.
+
+```bash
+docker compose up
+docker compose up -d # or run detached
+docker compose logs -t -f # view output from all containers
+```
+
+### MONAI Deploy Services
+- Informatics Gateway - http://localhost:5000 or http://172.29.0.50:5000
+  - SCP Port 104
+- Workflow Manager - http://localhost:5001 or http://172.29.0.60:5001
+- Task Manager - http://localhost:5002 or http://172.29.0.70:5002
+
+### Third-Party Services
+- Orthanc - http://localhost:8089 or http://172.29.0.100:5000
+  - SCP Port 1114
+- MinIO - http://localhost:9001 or http://172.29.0.10:9001
+- RabbitMQ - http://localhost:15672 or http://172.29.0.20:15672
+- MongoDB - http://localhost:27017 or http://172.29.0.30:27017
+
+## Running a MONAI Deploy Workflow
+
+This package includes Orthanc running and connected to the Informatics Gateway, with all required AE Titles pre-configured.
```

Comment:
NIT. Need to stress that the Orthanc instance is stood up purely for users convenience and for running the built-in example and it is NOT a component of MONAI Deploy. User has full control over what they deemed as suitable external DICOM devices.